website/docs/docs/dbt-versions/core-upgrade/01-upgrading-to-v1.8.md

```diff
@@ -82,6 +82,12 @@ models:
 
 The [`run`](/reference/commands/run#the-`--empty`-flag) and [`build`](/reference/commands/build#the---empty-flag) commands now support the `--empty` flag for building schema-only dry runs. The `--empty` flag limits the refs and sources to zero rows. dbt will still execute the model SQL against the target data warehouse but will avoid expensive reads of input data. This validates dependencies and ensures your models will build properly.
 
+### Spaces in dbt model names
+
+We will begin deprecating support for spaces in dbt model names in v1.8 (raising a warning) before removing support entirely in v1.9 (raising an error). Reasons for removing spaces in model names include:
+- Spaces in a model name makes in impossible to `--select` the model name because the argument gets split into pieces over spaces very early in the pipeline.
```

Comment:
```suggestion
- Spaces in a model name make it impossible to `--select` the model name because the argument gets split into pieces over spaces very early in the pipeline.
```